commons/_shared/bs-lib-components/ui/src/components/table/BsDSSTableFunctional.vue

```diff
@@ -61,7 +61,10 @@ export default defineComponent({
                     const data = this.transformDSSDataToQTableRow(val);
                     this.setFetchingChunk(false);
                     resolve(data);
-                }).catch(reject);
+                }).catch(err => {
+                    this.setFetchingChunk(false);
+                    reject(err);
```

Comment:
You are calling the function `this.setFetchingChunk(false);` in both cases. This is where the  `finally()` function is useful. It can be used along with `then(...).catch(...).finally(...)` and will be called whatever the promise was rejected or resolved